examples/auth0/next.config.js

```diff
@@ -0,0 +1,19 @@
+const dotenv = require('dotenv')
+const { withAuth0 } = require('@auth0/nextjs-auth0')
+
+dotenv.config()
+
+module.exports = withAuth0({
```

Comment:
@ijjk could you make sure this is not needed. `nextjs-auth0` could be changed to use the `browser` property in `package.json` to handle the particular bit.

Reply:
PR here: https://github.com/auth0/nextjs-auth0/pull/6

Reply:
Awesome

Reply:
Updated